gateways/adyen/request_builder.go

```diff
@@ -249,3 +252,19 @@ func addIfNonEmpty(value string, key string, data *map[string]string) {
 		(*data)[key] = value
 	}
 }
+
+// extractAdyenStreetFormat extracts adyen street format from generic street address
+//                          returns (streetNumber, streetName) format
+//                          If address does not have leading street number, will return ("", street)
+func extractAdyenStreetFormat(streetAddress string) (string, string) {
+	streetLocation := regexp.MustCompile("^\\d+\\s").FindStringIndex(streetAddress)
+	if streetLocation == nil {
+		return "", streetAddress
+	}
+
+	if len(streetAddress) > streetLocation[1] {
```

Comment:
what is this piece doing? Can we add uit tests for it?

Reply:
This is just incase we happen to get the address '123 ' we do not throw an indexing error. 

I'll look into adding unit tests